lib/guessing-game.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  GuessingGame: function GuessingGame(answer) {
+  this.answer = answer
+  this.win = false
+  this.guess = function(number) {
+    if (number < this.answer) { return 'low' }
+    if (number > this.answer) { return 'high' }
+    if (number == this.answer) { this.win = true; return 'correct' }
```

Comment:
If you are using multiple statements, I would put this on separate lines rather than all on 1